contracts/core/lib/auction-price-libraries/LinearAuctionLibrary.sol

```diff
@@ -0,0 +1,57 @@
+/*
+    Copyright 2018 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.4.24;
+
+import { SafeMath } from "zeppelin-solidity/contracts/math/SafeMath.sol";
+
+/**
+ * @title LinearAuctionLibrary
```

Comment:
IMO, we can remove the suffix `Library`. The other lib files do not have it. Thus, it would be consistent.